app.rb

```diff
@@ -64,7 +64,7 @@ class App < Sinatra::Application
        end
     end
 
-    before '/user/login' do
+    before '/admin/login' do
         if session[:admin_id]
```

Comment:
I already have a field called `is_admin` in my user table, wouldn't creating this method would override that and another thing is that for me to create this method i would to add it to my user file and i won't be able to use the session object

Reply:
Secondly, this method will not exist in the user file. It can be in the application_controller, which you can inherit from for all your controllers. or in a helper module, which is accessible to the views